nodeadm/test/e2e/run.sh

```diff
@@ -12,7 +12,7 @@ echo "done! Test image: $TEST_IMAGE"
 
 for CASE_DIR in $(ls -d test/e2e/cases/*); do
   CASE_NAME=$(basename $CASE_DIR)
-  printf "🧪 Testing $CASE_NAME..."
+  printf " Testing $CASE_NAME..."
```

Comment:
aw don't be like that

Reply:
i actually don't know why/when I lost this one 🤣 lemme fix that